scala-package/core/src/main/scala/org/apache/mxnet/module/BaseModule.scala

```diff
@@ -264,7 +276,8 @@ abstract class BaseModule {
           s"in mini-batches (${out.size})." +
       "Maybe bucketing is used?")
     )
-    val concatenatedOutput = outputBatches.map(out => NDArray.concatenate(out))
+    val oBT = outputBatches.transpose
+    val concatenatedOutput = oBT.map(out => NDArray.concatenate(out))
```

Comment:
Do you also need to dispose ```oBT``` here ?

Reply:
This create a soft copy, the content inside contains same reference